package.json

```diff
@@ -27,7 +27,7 @@
     "@types/express": "^4.0.33",
     "@types/mime": "0.0.29",
     "@types/mz": "0.0.29",
-    "@types/node": "^4.0.30",
+    "@types/node": "=4.0.30",
```

Comment:
Hm yes correct, for some reason I found this solution in a different package that was affected by this regression. I did not check the docs. Anyways, #155 solves the issue so I will need to rebase after that.